server/sonar-web/src/main/js/apps/metrics/form-view.js

```diff
@@ -40,10 +40,10 @@ export default ModalForm.extend({
         callback({ id: value, text: value });
       },
       query (options) {
-        const items = that.options.domains.filter(function (d) {
+        const items = that.options.domains.filter(d => {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove curly braces and "return" from this arrow function body. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)